### ISSUE TITLE
Expose the train/split ratio in the model config

### ISSUE THREAD

**Issue by shuttie** [metarank/metarank#551]
Now it's hardcoded on 80%, which is fine for most of the cases, but not nice.

**Comment by nikdon** [comment]
I believe this one is resolved with #648?

**Comment by shuttie** [comment]
I guess there's a more generic ticket exists: https://github.com/metarank/metarank/issues/664

Currently, metarank just randomly samples click-through sessions into either train or validation set (with some hardcoded probability, like 80%). This is the most simple way to do it in MVP, but it has some drawbacks:
* the NDCG value on the test dataset is not going to match the one you get in prod, as the current splitting process leaks time into the training data. So you may train on click-through events from the future to predict the past.

We're thinking on adding a couple of more real-life splitting strategies, for example:
* time splitting: all click-throughs are sorted by time, and then split linearly into train/test.
* leave-last-out: within a visitor session, we sort click-throughs by time and assign first ones to train, and last ones to test.
* randomize: the current implementation
* randomize-sessions: group click-through by user session, and assign randomly whole sessions.

There are probably more strategies exist, but these ones are the most cited in the papers.
